doc/nrf/app_dev/device_guides/wifi_coex.rst

```diff
@@ -70,25 +69,6 @@ The following are the common requirements to use coexistence based on the MPSL C
    When using the nRF5340, apply steps 1 and 2 only to the network core.
    See :ref:`ug_multi_image`.
 
-.. _ug_radio_coex_bluetooth_only_based:
-
-MPSL provided Bluetooth-only coexistence
-========================================
-
-The MPSL-provided Bluetooth-only coexistence can be used only with the :ref:`ug_radio_coex_bluetooth_only_1wire` implementation.
-
-
-It is based on :ref:`nrfxlib:bluetooth_coex` provided by the :ref:`nrfxlib:mpsl` (MPSL) library.
-
-
-Supported implementations
-*************************
-
-The following are the implementations supported by the MPSL-provided Bluetooth-only coexistence.
-
-.. note::
-   Do not enable Wi-Fi coexistence on the nRF5340 SoC in conjunction with Coded Phy and FEM, as this can lead to undefined behavior.
-
 .. _ug_radio_mpsl_cx_nrf700x:
 
 nRF70 Series Wi-Fi coexistence
```

Comment:
The following subsections are only providing `Refer to XYZ for general requirements` and don't state the use case for the given coexistence scenario. Please add these use cases.

Reply:
can you be more specific?